frontend/src/components/search/Search.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import {DataSearch, ReactiveList, MultiDropdownList, SingleDropdownRange} from "@appbaseio/reactivesearch";
+import {parseDate} from "../../utils/common";
+
+import DatasetCard from "../datasets/DatasetCard";
+import {Grid} from "@mui/material";
+import Layout from "../Layout";
+import {FilesTableFileEntry} from "../files/FilesTableFileEntry";
+import { SearchResult } from "./SearchResult";
+
+
+export function Search() {
+	// @ts-ignore
+	return (
+		<Layout>
+			<div className="outer-container">
+				<Grid container spacing={4}>
+					<Grid item xs>
+						{/*search*/}
+						<DataSearch
+							title="Search for Datasets and Files"
+							componentId="searchbox"
+							autosuggest={true}
+							highlight={true}
+							queryFormat="or"
+							fuzziness={0}
+							debounce={100}
+							react={{ and: ["creatorfilter",
+									"downloadfilter",
+									"modifyfilter"]}}
+							// apply react to the filter
+							URLParams={true}
+							showFilter={true}
+							showClear
+							renderNoSuggestion="No suggestions found."
+							dataField={[
+								{field: "name", weight: 3},
+								{field: "description", weight: 2},
+								{field: "author.keyword", weight: 1},
+								{field: "creator.keyword", weight:1}
+								]}
+							// placeholder="Search for Dataset"
+							innerClass={{
+								title: "search-title",
+								input: "search-input",
+							}}
+						/>
+						{/*filters*/}
+						<Grid container spacing={2} sx={{marginBottom: "20px"}}>
```

Comment:
I am not seeing the author filter (see screenshot below). This happened before and after I dropped the indexes and rebuilt them with new data. Now with the new data I am not getting any results. Trying to debug.
![Screen Shot 2022-10-24 at 2 06 24 PM](https://user-images.githubusercontent.com/599725/197605951-a465cfb9-1d4f-49e9-b4fc-15541f965820.png)

Reply:
could you double check in the dejavu you see author as a keyword? 
<img width="733" alt="image" src="https://user-images.githubusercontent.com/13950475/197606824-37ef49b3-7966-428b-a84a-82e40e9092a3.png">

Reply:
It seeing it as of type text with fields keyword.

![Screen Shot 2022-10-24 at 4 21 25 PM](https://user-images.githubusercontent.com/599725/197631961-1f1d78c3-f1d0-42e5-9550-17ef59cc911b.png)
![Screen Shot 2022-10-24 at 4 20 42 PM](https://user-images.githubusercontent.com/599725/197631971-f106eabe-16d4-4389-a26a-a709627adaef.png)

Reply:
Mapping definition in backed looks ok

<img width="416" alt="Screen Shot 2022-10-24 at 4 23 05 PM" src="https://user-images.githubusercontent.com/599725/197632252-9ef54d60-8477-4cf0-a96a-83234c010d29.png">

Reply:
hmm my guess is you might have hybrid records (some are text others are keyword?). Could you try delete all the elasticsearch records and start all over? (Or just delete the volumn)

Reply:
I dropped both indexes and created a bunch of new datasets using the `populate_fake_data.py` script. I'll try again.

Reply:
I made a PR with some simple script to drop indexes and mongodb https://github.com/clowder-framework/clowder2/pull/161/files.

Reply:
Sure. I am wondering when this issue will arise because I am creating the indices when the app starts. Unless someone deletes those indices, this should never arise. What do you suggest?

Reply:
I am confused now. It is now working with just one dataset entry and no files. We can merge and keep testing. @ddey2 can you do a couple of tests starting from scratch? Main has a script to drop indexes and some mongo tables https://github.com/clowder-framework/clowder2/blob/main/scripts/develop/delete-data.sh.

Reply:
Sure.